extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -96,7 +99,14 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 	};
 
 	GoogleTag.prototype.push = function (callback) {
-		googleTag.cmd.push(callback);
+		if (this.storeCommands) {
+			cmd.push(callback);
+		}
+		window.googletag.cmd.push(callback);
```

Comment:
Yup, callback has to go to googletag because we don't want to affect non blocking users. On the other hand we need those callbacks when we want to run recovery (googletag.cmd is cleared for AdBlock Plus users). Now I see it's better to rename this temporary cmd array - maybe `recoveryCmd`?